ulib/axstarry/src/ctypes.rs

```diff
@@ -4,6 +4,8 @@ use axhal::{
 };
 use bitflags::*;
 use core::panic;
+/// a flag used in sys_dup3
```

Comment:
You can wrap the flags as a full type for dup3 syscall in the next commit

Reply:
ok